.github/workflows/main-build.yml

```diff
@@ -0,0 +1,121 @@
+name: Main Build
+
+on:
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  integration-test:
+    runs-on: ubuntu-20.04
+    name: Integration Test - ${{ matrix.name }}
+    strategy:
+      matrix:
+        include:
+# TODO(anuraaga): Enable after opentelemetry-lambda changes applied.
+#          - name: Java AWS SDK (Agent)
+#            language: java
+#            build_directory: opentelemetry-lambda/java
+#            build_command: ./gradlew build
+#            terraform_directory: opentelemetry-lambda/java/integration-tests/aws-sdk/agent
+#          - name: Java AWS SDK (Wrapper)
+#            language: java
+#            build_directory: opentelemetry-lambda/java
+#            build_command: ./gradlew build
+#            terraform_directory: opentelemetry-lambda/java/integration-tests/aws-sdk/wrapper
+#          - name: Java OkHttp (Wrapper)
+#            language: java
+#            build_directory: opentelemetry-lambda/java
+#            build_command: ./gradlew build
+#            terraform_directory: opentelemetry-lambda/java/integration-tests/okhttp/wrapper
+          - name: NodeJS AWS SDK
+            language: nodejs
+            build_directory: opentelemetry-lambda/nodejs
+            build_command: npm install
+            terraform_directory: opentelemetry-lambda/nodejs/integration-tests/aws-sdk
```

Comment:
For canary test, terraform_directory will be `sample-apps`

Reply:
Why canary test have to change directory to smple-apps? cannot use integration-tests?

Reply:
we can use 
```
cat release-note.md | grep $region | sed "s/[ ]//g;s/\|//g" | sed "s/^$region//"
```
to extract the layer ARN, and set it as an input parameter when building sample application for canary.
See an example of [release-note.md](https://github.com/wangzlei/horn/blob/main/adot/python/release-note.md)